### PR TITLE
Fix agents

### PR DESCRIPTION
**Pull request by Red-Pheonix**
Some of the rl agents were not working(`frap`, `mplight`, `colight`, `ppo`, `magd` were not working when I tested). I made some adjustmens to make `frap`, `mplight` and `colight` work for now.

For `frap` and `mplight`, in newer versions of numpy, there is a deprecation error when using `np.int`. They now use `np.int64` instead. `mplight` used to give errors when using networks with irregular lane numbers(like `cologne3`), it now pads its input deal with this problem.

For `colight`, the graph interface can now load directly from sumo network files. So `colight` can now directly work with `sumo` networks without using converters. In addition, like with `mplight`, it now pads its input to deal with irregular shapes.

### CODE REVIEW

File: agent/frap.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def relation(self):
         '''
         comp_mask = []
         for i in range(len(self.phase_pairs)):
-            zeros = np.zeros(len(self.phase_pairs) - 1, dtype=np.int)
```
Comment (wingsweihua) [inline agent/frap.py:141]: Hi @Red-Pheonix, can you make sure this modification is compatible with previous numpy versions as well? An easy fix could be adding a checker on numpy version with if/else - but if you have better solutions, we would really appreciate it!

File: agent/mplight.py [hunk 2]
```diff
@@ -215,7 +215,7 @@ def relation(self):
         '''
         comp_mask = []
         for i in range(len(self.phase_pairs)):
-            zeros = np.zeros(len(self.phase_pairs) - 1, dtype=np.int)
+            zeros = np.zeros(len(self.phase_pairs) - 1, dtype=np.int64)
```
Comment (wingsweihua) [inline agent/mplight.py:218]: Same comment - compatible with previous versions.

**Review by wingsweihua — COMMENTED** [review]
Hey @Red-Pheonix Thanks for the update! I left my comment in-line. Please take a look and let me know if you have any question.

File: agent/frap.py (hunk 1 above)
Comment (Red-Pheonix) [reply agent/frap.py:141]: The fix I added should also work in older versions in numpy. Going by the `requirements.txt` file, `np.int64` is also available in `numpy version 1.17`. So no checks are needed.

Comment (Red-Pheonix) [reply agent/frap.py:141]: Here is the documentation page listing `np.int64` for `numpy version 1.17`  https://numpy.org/doc/1.17/user/basics.types.html

Comment (wingsweihua) [reply agent/frap.py:141]: Thanks!

**Review by wingsweihua — APPROVED** [review]

**Pull request merged**